commands/template.go

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) OpenFaaS Author(s) 2018. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package commands
+
+import (
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	faasCmd.AddCommand(templateCmd)
+}
+
+// templateCmd allows access to store and pull commands
+var templateCmd = &cobra.Command{
+	Use:   `template [COMMAND]`,
+	Short: "OpenFaaS template store and pull commands",
+	Long:  "Allows browsing templates from store or pulling custom templates",
+	Example: `  faas-cli template pull https://github.com/custom/template
+  faas-cli template store list
+  faas-cli template store ls
+  faas-cli template store pull ruby-http
```

Comment:
The docs say that multiple template/repo names are possible here, but the help text doesn't represent that same data.

I'd prefer us to only allow one repo/template to be pulled at a time using the CLI.

Reply:
ok I will remove that logic but should I do this in new commit, I think I can force push in this one ?

Reply:
A new commit addressing a single comment is preferable because it is easier to then track. Then rebase and squash once at the end after it has been fully approved.

Reply:
Ok thanks again @LucasRoesler I will proceed with that approach from now on

Reply:
Removed the feature ✔️